docs/docs/latest/topics/running/ingress-controller.md

```diff
@@ -13,6 +13,8 @@ If you're new to the Ambassador Edge Stack and to Kubernetes, we'd recommend you
    In Kubernetes 1.13 and below, the `Ingress` was only included in the `extensions` api.
 
    Starting in Kubernetes 1.14, the `Ingress` was added to the new `networking.k8s.io` api.
+   
+   Kubernetes 1.18 introduced the `IngressClass` resource to the existing `networking.k8s.io/v1beta1` api.
```

Comment:
Ideally, I would include a "Read more ..." link here pointing to our blogpost with changes to the Ingress API in k8s 1.18 and how to use the IngressClass resource in Ambassador.
However, this blog entry does not exist yet.

Also, since the note bellow is still valid:
> Since both are still supported in all 1.14+ versions of Kubernetes, this document will use `extensions/v1beta1` for compatibility reasons.

we should wait for IngressClass and k8s 1.18+ is widely adopted before updating the samples from this reference doc.

Reply:
Are we publishing a new blog post or updating the existing one (or both)?